features/step_definitions/api-token.js

```diff
@@ -167,4 +170,17 @@ module.exports = function server() {
     this.Then(/^the token's 'last used' property will not be updated$/, () => {
         Assert.strictEqual(this.updatedToken.lastUsed, this.testToken.lastUsed);
     });
+
+    this.When(/^they revoke the token$/, () =>
+        request({
+            uri: `${this.instance}/${this.namespace}/tokens/${this.testToken.id}`,
+            method: 'DELETE',
+            auth: {
+                bearer: this.jwt
+            }
+        }).then(response => Assert.strictEqual(response.statusCode, 204)));
+
+    this.Then(/^the login attempt fails$/, () => {
+        Assert.strictEqual(this.loginResponse.statusCode, 401);
```

Comment:
But where are you calling log in in that step then? I'm only seeing the POST to logout